sdk/core/System.ServiceModel.Rest/src/RequestBody.cs

```diff
@@ -12,6 +14,13 @@ namespace System.ServiceModel.Rest.Core
     /// </summary>
     public abstract class RequestBody : IDisposable
     {
+        /// <summary>
+        /// Creates an instance of <see cref="RequestBody"/> that wraps a <see cref="Stream"/>.
+        /// </summary>
+        /// <param name="stream">The <see cref="Stream"/> to use.</param>
+        /// <returns>An instance of <see cref="RequestBody"/> that wraps a <see cref="Stream"/>.</returns>
+        public static RequestBody CreateFromStream(Stream stream) => new StreamBody(stream);
```

Comment:
This has this name to prevent versioning issues with the Utf8JsonRequestContent type in shared source.  We can consolidate this and unify the API when we move Azure libraries to use the corresponding public type and remove the shared source file.